### PR TITLE
Add utilities to connect to or create kopia repository

### PR DESCRIPTION
**Pull request by ankitjain235**
## Change Overview

This PR add utilities to create a Kopia repository or connect to existing ones.

## Pull request type

Please check the type of change your PR introduces:
- [ ] :construction: Work in Progress
- [ ] :rainbow: Refactoring (no functional changes, no api changes)
- [ ] :hamster: Trivial/Minor
- [ ] :bug: Bugfix
- [ ] :sunflower: Feature
- [ ] :world_map: Documentation
- [ ] :robot: Test

## Issues 

- fixes #issue-number

## Test Plan

- [ ] :muscle: Manual
- [ ] :zap: Unit test
- [ ] :green_heart: E2E

### CODE REVIEW

File: pkg/kopia/repository/create.go [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+	"github.com/kanisterio/kanister/pkg/log"
+	"github.com/kanisterio/kanister/pkg/utils"
+)
+
+const (
+	// Kopia returned errors
+	errCodeOutOfMemory = "command terminated with exit code 137"
+	errAccessDenied    = "Access Denied"
+	errInvalidPassword = "invalid repository password"
+)
```
Comment (shlokc9) [inline pkg/kopia/repository/create.go:25]: These error messages are already present in `pkg/kopia/errors/errors.go`
Can we make use of those?

File: pkg/kopia/repository/create.go [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+	keepMonthly = "--keep-monthly"
+	keepAnnual  = "--keep-annual"
+
+	// Compression
+	compressionAlgorithm = "--compression"
+
+	// Compression Algorithms recognized by Kopia
+	s2DefaultComprAlgo = "s2-default"
+)
+
```
Comment (shlokc9) [inline pkg/kopia/repository/create.go:107]: These constant flags can be moved to `github.com/kanisterio/kanister/pkg/kopia/command/const.go`

**Review by shlokc9 — CHANGES_REQUESTED** [review]
Suggested some minor changes

**Comment by ankitjain235** [comment]
The workflow has been changed and separate PRs have been raised. Closing this PR.

**Pull request closed without merging**